### PR TITLE
icons for sample types

### PR DESCRIPTION
**Pull request by ritch**
Screen Shot 2022-08-29 at 3 16 21 PM

Screen Shot 2022-08-29 at 3 25 47 PM

Screen Shot 2022-08-29 at 3 26 15 PM

### CODE REVIEW

File: app/packages/app/src/components/ImageContainerHeader.tsx [hunk 1]
```diff
@@ -117,4 +120,20 @@ const ImageContainerHeader = () => {
   );
 };
 
+function MediaTypeIcon({ type }) {
+  const style = {
+    position: "relative",
+    top: "0.5rem",
+  };
```
Comment (benjaminpkane) [inline app/packages/app/src/components/ImageContainerHeader.tsx:127]: Unused, maybe?

File: app/packages/app/src/components/ImageContainerHeader.tsx [hunk 2]
```diff
@@ -54,35 +60,32 @@ const SliderContainer = styled.div`
...
 const ImageContainerHeader = () => {
+  const type = useRecoilValue(fos.dataset).mediaType;
   const setGridZoom = useSetRecoilState(gridZoom);
   const gridZoomRangeValue = useRecoilValue(gridZoomRange);
   const theme = useTheme();
   const group = useRecoilValue(isGroup);
+  const element = useRecoilValue(fos.elementNames);
+  const total = useRecoilValue(
+    fos.count({ path: "", extended: false, modal: false })
+  );
```
Comment (benjaminpkane) [inline app/packages/app/src/components/ImageContainerHeader.tsx:72]: This is going to suspend, which will yield a bad loading state. It should be used in a component within a `Suspense` like `Count` was previously.

File: app/packages/app/src/components/ImageContainerHeader.tsx [hunk 3]
```diff
@@ -54,35 +60,32 @@ const SliderContainer = styled.div`
...
   return (
     <SamplesHeader>
       <GridActionsRow />
       <RightContainer>
         <Suspense fallback={<RightDiv>{"Loading..."}</RightDiv>}>
-          <Count />
+          <PillButton
+            icon={<MediaTypeIcon type={type} />}
+            text={<PathEntryCounts modal={false} path={""} />}
+            title={`${total.toLocaleString()} ${_.capitalize(type)} ${
```
Comment (benjaminpkane) [inline app/packages/app/src/components/ImageContainerHeader.tsx:82]: I don't think the type needs to be capitalized. The values are not capitalized in Python, e.g.:

```
Name:        quickstart
Media type:  image
Num samples: 200
Persistent:  False
Tags:        []
Sample fields:
    id:           fiftyone.core.fields.ObjectIdField
    filepath:     fiftyone.core.fields.StringField
    tags:         fiftyone.core.fields.ListField(fiftyone.core.fields.StringField)
    metadata:     fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.metadata.ImageMetadata)
    ground_truth: fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.labels.Detections)
    uniqueness:   fiftyone.core.fields.FloatField
    predictions:  fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.labels.Detections)
```

**Review by benjaminpkane — CHANGES_REQUESTED** [review]
Looks good. I like the orange, but it may be deceiving as it is not a button. Maybe something darker?

**Comment by brimoor** [comment]
We decided not to add icons right now

**Pull request closed without merging**